Content.Shared/Slippery/SlipperySystem.cs

```diff
@@ -75,7 +75,10 @@ private void TrySlip(EntityUid uid, SlipperyComponent component, EntityUid other
             _physics.SetLinearVelocity(other, physics.LinearVelocity * component.LaunchForwardsMultiplier, body: physics);
 
             if (component.SuperSlippery)
-                EnsureComp<SlidingComponent>(other);
+            {
+                var sliding = EnsureComp<SlidingComponent>(other);
+                sliding.CollidingEntities.Add(uid);
+            }
         }
 
```

Comment:
If you're not colliding with what slipped you won't this hang around indefinitely.

Reply:
StepTrigger makes it so you won't start collision if you already stepped on it, but the ending of the collision is not blocked. So it will get properly removed from the list when collision ends.

Reply:
Add a debug assert then.

Reply:
what should I be asserting?

Reply:
I added a debug assert that checks if the list is empty after adding the slidingcomponent

Reply:
The debug assert you've added doesn't make sense as it's never possible for it to be true, you should be checking that the slidingcomponent's contacts even have the entity.

Reply:
to start sliding you have to collide with the entity, so won't that also be always true? I'll add it though